episodes/quality-control-and-manual-segmentation.md

```diff
@@ -0,0 +1,547 @@
+values of 0, 1, 2 and 3.
+
+Let's take a look at the pixel values in the instance segmentation:
+
+- Click the ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/visibility.svg
+){alt="A screenshot of Napari's eye button" height='30px'} icon next to 
+'instance_seg' in the layer list to make it visible. 
+- Click the ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/visibility.svg
+){alt="A screenshot of Napari's eye button" height='30px'} icon next to 
+'semantic_seg' in the layer list to hide it.
+- Make sure the 'instance_seg' layer is selected in the layer list. It should be 
+highlighted in blue.
+
+If you hover over the segmentation now, you should see that each nucleus has a 
+different pixel value. We can find the maximum value via the console like so:
+
+```python
+# Get the instance segmentation data
+image = viewer.layers['instance_seg'].data
+
+# Print the maximum pixel value
+print(image.max())
+
+```
+
+```output
+19
+
+```
+
+Instance segmentations tend to have many more pixel values, as there is one per 
+object 'instance'. When working with images with very large numbers of cells, 
+you can quickly end up with hundreds or thousands of pixel values in an instance 
+segmentation.
+
+So to summarise - segmentations are just images where the pixel values represent 
+specific categories, or individual instances of that category. You will 
+sometimes see them referred to as 'label images', especially in the context of 
+Napari. Segmentations with only two values (e.g. 0 and 1 for background vs some 
+specific category) are often referred to as 'masks'.
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Segmentation shapes and data types
+
+Copy and paste the following into Napari's console:
+
+```python
+nuclei = viewer.layers["nuclei"].data
+semantic_seg = viewer.layers["semantic_seg"].data
+instance_seg = viewer.layers["instance_seg"].data
+```
+
+- What shape are the `nuclei`, `semantic_seg` and `instance_seg` images?
+
+- What pattern do you see in the shapes? Why?
+
+- What bit-depth do the `semantic_seg` and `instance_seg` images have?
+
+- If I want to make an instance segmentation of 100 nuclei - what's the 
+minimum bit depth I could use? (assume we're storing unsigned integers) Choose 
+from 8-bit, 16-bit or 32-bit.
+
+- If I want to make an instance segmentation of 1000 nuclei - what's the 
+minimum bit depth I could use? (assume we're storing unsigned integers) Choose 
+from 8-bit, 16-bit or 32-bit.
+
+Recall that we looked at image shapes and bit-depths in the ['What is an image?' 
+episode](what-is-an-image.md).
+
+:::::::::::::::::::::::: solution 
+
+### Solution
+
+### Shapes + pattern
+
+```python
+print(nuclei.shape)
+print(semantic_seg.shape)
+print(instance_seg.shape)
+```
+
+```output
+(60, 256, 256)
+(60, 256, 256)
+(60, 256, 256)
+```
+
+All the images have the same shape of (60, 256, 256). This makes sense as 
```

Comment:
Is there any scope here for highlighting just because they are the same image size, you might think that they could require the same hard disk storage, but in many cases these mask/segmentation images can be compressed heavily without loss of information.